README.md

```diff
@@ -72,5 +72,18 @@ the given file into your `.git/hook` directory:
 
     cp ./_devtools/git-hooks/post-commit .git/hooks/
 
+## Custom liquid tags
+
+Custom liquid tags are used to make linking to javadocs, properties, and documents easier
```

Comment:
Could add a sentence mentioning/linking to source.

Reply:
Fixed